src/PackageServiceProvider.php

```diff
@@ -32,6 +32,11 @@ public function register()
      */
     public function boot()
     {
+        //Register commands
+         $this->commands([
+            Console\Commands\Install::class,
+        ]);
```

Comment:
Should we add the Uninstall command here too?

Reply:
Thanks @ryancooley, I missed that. I already pushed the changes